### PR TITLE
adding ChronAm Title Essay Datasets

### PR DESCRIPTION
**Pull request by danieljohnevans**

### CODE REVIEW

**Comment by ptrourke** [comment]
Can you supply the link for "Read more about `unigrams` and `word tokenization` here:" (analysis notebook, after code block 5)?

**Comment by ejakeway** [comment]
Hi Daniel, thanks for this work! I concur with Patrick on filling in the missing link. My only other suggestion (which you can take or leave depending on effort involved) would be to rename the output csv files so it's clearer which one is the full set of ChronAm title essays and which one is the result of the NLP analysis.

**Review by ejakeway — COMMENTED** [review]
Review complete; see comment above.

**Comment by ejakeway** [comment]
Sorry--one more thing. Would you please add your short bio, dates, and contact info for a point person on the ChronAm team to the project overview section in the README? Thanks!

**Comment by danieljohnevans** [comment]
Thanks for feedback folks! I think I made necessary fixes. Let me know.

File: loc.gov JSON API/Chronicling_America_Title_Essay_Datasets/download.ipynb [hunk 1]
```diff
@@ -0,0 +1,2007 @@
...
+   "source": [
+    "---------"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "79cf4c70",
+   "metadata": {},
+   "source": [
+    "Next, we will take that list of URLs and create a small script to pull down data from the Library API. I originally wrote it as a parallelized script that downloaded the code in about 45 minutes. Unfortunately, it kept overloading the Library of Congress API. I re-wrote it so that it instead uses a handful of 'user-agents' to download the data. This \"slow\" method takes about three hours. "
```
Comment (ptrourke) [inline loc.gov JSON API/Chronicling_America_Title_Essay_Datasets/download.ipynb:304]: We'd appreciate it if you removed this section on using multiple user agents (Part 2 first markdown cell, ll. 266-306,  and input 11 code cell through input 13 code cell, ll. 479-539), replace the call for `rando_headers()` in l. 1526 with a note on how to set a custom header, and increased the value in `time.sleep(0.8)` in l. 1548 to match the rate limits on https://www.loc.gov/apis/json-and-yaml/ for the newspapers endpoint.

**Review by ptrourke — CHANGES_REQUESTED** [review]

File: loc.gov JSON API/Chronicling_America_Title_Essay_Datasets/download.ipynb (hunk 1 above)
Comment (danieljohnevans) [reply loc.gov JSON API/Chronicling_America_Title_Essay_Datasets/download.ipynb:304]: Thanks for comments! Should be resolved. Let me know if this fixes issues.

**Review by ptrourke — APPROVED** [review]

**Pull request merged**